docs/enterprise-guide/serialization.md

```diff
@@ -51,12 +51,15 @@ Currently, you can only load dumps into a Metabase instance that were created fr
 
 `java -jar metabase.jar load [my_dump] --mode [skip/update] --on-error [continue/abort]`
 
-The `--mode` flag lets you specify what to do when encountering a duplicate dashboard, question, or any Admin Panel settings that already set (again, except for permissions and user settings, which are not currently included in data dumps). It can either `skip` that item and do nothing to it, or `update` it with the version being loaded. The default is `skip`.
+The `--mode` flag lets you specify what to do when encountering a duplicate dashboard, question, or any Admin Panel settings that already set (again, except for permissions and user settings, which are not currently included in data dumps). It can either `skip` that item and do nothing to it, or `update` it with the version being loaded. The default is `skip`. For example, if you change the order, add or remove any cards in a dashboard that has been loaded previously, mode `skip` won't do any modifications to it, as the load process "skips" any object that previously exists no matter if there are changes on the source file.
 
 The `--on-error` flag allows you to specify whether the load process should keep going or stop when there's an error. The default is `continue`. Note that `abort` won't undo any successful artifact loads that happened before an error was encountered.
 
 Both of these flags are optional.
 
+### Current limitations
+
+- The serialization feature is not able to identify a dashboard card in the destination if the visualization settings in the source dashboard have changed
```

Comment:
Is this what you mean?

```suggestion
-  If the only change for a dashboard card is its visualization settings (that is, the visualization settings on the dashboard card, not the visualization for the card's question), the serialization process won't pick up that change, which means the serialization process won't apply the card's new dashboard visualization setting to the dashboard card at the destination.
```

And is there a workaround?

Reply:
in fact it will duplicate the card on destination, but Johannes does not agree with adding this section so we should delete it